src/rdpc/variantCallingAnalysesProcessor.ts

```diff
@@ -187,15 +187,22 @@ const convertAnalysis = (
     const infoMapPerAnalysis = analysis.donors.reduce<
       StringMap<SangerAndMutectInfo>
     >((infoAccumulator, donor) => {
-      const workflowName = analysis.workflow.workflowName.toLocaleLowerCase();
+      let workflowName = "";
+      if (analysis.workflow && analysis.workflow.workflowName) {
+        workflowName = analysis.workflow.workflowName.toLocaleLowerCase();
+      } else {
+        logger.warn(`Incomplete RDPC Data: analysis id: ${analysis.analysisId} does not have 'workflow' or 'workflowname',
+        this analysis will not be indexed.`);
```

Comment:
We are still indexing this analysis, just putting blank workflow.

Reply:
After discussion, without the workflow name we can't adjust any stats for a donor from this workflow. Lets rephrase this to say "this analysis won't be included in the donor's aggregated stats"